inference-engine/src/low_precision_transformations/src/split.cpp

```diff
@@ -83,7 +83,8 @@ bool SplitTransformation::transform(TransformationContext& context, ngraph::patt
             parent = subtract;
         }
 
-        const auto multiply = std::make_shared<DequantizationMultiply>(parent, splitedMul[i]);
+        const auto multiply = std::make_shared<op::TypeRelaxed<DequantizationMultiply>>(parent, splitedMul[i]);
+        NetworkHelper::setOutDataPrecisionForTypeRelaxed(multiply, dequantization.multiply->get_output_element_type(0));
```

Comment:
Can we use `deqPrecision` instead of `dequantization.multiply->get_output_element_type(0)`?

Reply:
Ok, thanks!